### ISSUE TITLE
Custom Smileys Window

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dequis/msn-pecan#107]
```
Hello, i recently switched from my ubuntu-linux box to a windows only
machine. With the last pecan 0.0.17 i can't get custom smileys send to
work. Everything works fine under linux, i'm able to see the custom smileys
directly in the smilies window, while with win32 i can just see the
in tools=>smileys but neither in the chat nor in the chat smileys-button
window.
On win32 i can't just see the incoming custom, but not sending them in any way.

Upgrading to pidgin 2.5.3 doesn't change anything.
```

Original issue reported on code.google.com by `markuznightwind@gmail.com` on 25 Dec 2008 at 4:42

**Comment by GoogleCodeExporter** [comment]
```
I think I know the issue. I need to recompile my win32 libpurple.
```

Original comment by `felipe.contreras` on 29 Dec 2008 at 9:22
- Changed state: **Accepted**
- Added labels: **Type-Defect**, **Milestone-0.1.0**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Back from vacations. Thanks for the answer, hope it will be done for 0.18 
version, so
i can use pecan even on windows :)

```

Original comment by `markuznightwind@gmail.com` on 12 Jan 2009 at 1:48

**Comment by GoogleCodeExporter** [comment]
```
I'm also waiting for update eagerly
```

Original comment by `milo...@gmail.com` on 13 Jan 2009 at 8:06

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed in 0.0.18 already.

Please verify.
```

Original comment by `felipe.contreras` on 15 Mar 2009 at 7:44
- Changed state: **Fixed**
